src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,10 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given **thread**. 
+
+Read how to [list all available chats](/management/configuration-api/v2.0#list-all-tags) and [create a new tag](/management/configuration-api/v2.0#add-a-tag). **Tags** are case sensitive in this method.
```

Comment:
How about we move the info about case sensitivity to the Request table to comply with the format proposed in [this PR](https://github.com/livechat/livechat-public-docs/pull/1145/files#diff-26505ab33f8366eff9fa237c91538cc37a2f61c11cdd44e89ba26aeb8779d7b5R2492)?
<img width="648" alt="image" src="https://user-images.githubusercontent.com/42515529/167863750-44565220-2305-4940-87f2-83465879551b.png">

Reply:
Yes, I like the idea! Just to confirm - all the places, including v3.5?